components/validateur-report/alerts-header.js

```diff
@@ -0,0 +1,35 @@
+import PropTypes from 'prop-types'
+import {Pane, WarningSignIcon, BanCircleIcon, MapMarkerIcon, InfoSignIcon} from 'evergreen-ui'
+
+function AlertHeader({isVoie, hasErrors, hasWarnings, hasInfos, children}) {
+  const hasVoieNoAlerts = !hasWarnings && !hasErrors && !hasInfos
+
+  return (
+    <Pane display='flex' gap={8} alignItems='center'>
+      <Pane display='flex' gap={5} >
+        {hasErrors && <BanCircleIcon color='red500' size={isVoie ? 22 : 16} />}
+        {hasWarnings && <WarningSignIcon color='orange500' size={isVoie ? 22 : 16} />}
+        {hasInfos && <InfoSignIcon color='blue500' size={isVoie ? 22 : 16} />}
+        {hasVoieNoAlerts && <MapMarkerIcon size={22} />}
+      </Pane>
```

Comment:
Pour rendre la logique du rendu plus simple à comprendre, je serais parti là-dessus. Comme aucun des autres `has…` ne peuvent apparaitre si `hasVoieNoAlerts ` est vrai.

```suggestion
    <Pane display='flex' gap={5} >
      {hasVoieNoAlerts ? <MapMarkerIcon size={22} /> : (
        <>
          {hasErrors && <BanCircleIcon color='red500' size={isVoie ? 22 : 16} />}
          {hasWarnings && <WarningSignIcon color='orange500' size={isVoie ? 22 : 16} />}
          {hasInfos && <InfoSignIcon color='blue500' size={isVoie ? 22 : 16} />}
        </>
      )}
    </Pane>
```

Attention également, `isVoie ? 22 : 16` est dupliqué inutilement.